.github/workflows/app_build.yml

```diff
@@ -15,12 +15,12 @@ env:
   BUILD_TESTING: ON
   BUILD_BENCHMARK: ON
   BUILD_PACKAGE: ON
-  QT_VERSION: 6.6.3
+  QT_VERSION: 6.5.3
 
 jobs:
   build:
-    # Note: as of 2021-01-29, this only works for push, not for pull request
-    # if: "!(contains(github.event.head_commit.message, 'skip') && contains(github.event.head_commit.message, 'ci'))"
+    if: ${{ github.event_name == 'push' || !github.event.pull_request.draft }}
```

Comment:
@jmarrec what do you think about this?  I can't remember when we would want to use 'skip' or 'ci' instead?

Reply:
Hmm the way I have it now marking as "Ready to Review" doesn't trigger the build, I'll work on that.

Reply:
Ok now it runs when marked ready for review

Reply:
So you can push a commit with "[skip ci]" in the commit message and the run won't be triggered.

But this is now baked into github actions anyways; https://docs.github.com/en/actions/managing-workflow-runs/skipping-workflow-runs